scripts/dockerbot/pytestbase/setup.sh

```diff
@@ -216,7 +219,16 @@ run_n_log "find $CFME_REPO_DIR -name \"*.pyc\" -exec rm -rf {} \;"
 
 set +e
 
+if [ "$USE_SPROUT" = "yes" ];
+then
+    miq sprout checkout --populate-yaml  2>&1 >> $ARTIFACTOR_DIR/setup.txt &
```

Comment:
The miq sprout command has its own provision timeout, is the bit below to fail hard after 5 seconds? That isn't enough time for it to spin up and appliance if it doens't have one in the pool. I think we should let it run until the provisioning timer runs out.

Reply:
it uses the do or die with 5 attempts each 10 second sleep, we can up that if necessary - for now it worked fine

to me its not clear what a good time-out would be
for the python implementation i'd opt for allowing the checkout command to fail/exit, and the waiting for the data to break out only if the command exits with a failed checkout

Reply:
right, I think we push it up for a 5 min max

Reply:
it was bumped from 5 10 to 5 60